utils/ChromiumDownloader.js

```diff
@@ -149,13 +152,22 @@ module.exports = {
       executablePath,
       folderPath,
       downloaded: fs.existsSync(folderPath),
-      url: util.format(downloadURLs[platform], revision)
+      url: util.format(getDownloadUrl(platform), revision)
```

Comment:
you can remove the `url` field: it's never used (and will be misleading with this change)

Reply:
ok 👍